### PR TITLE
feat: Breaking the Monolith Failed! Where is my data?

### PR DESCRIPTION
**Pull request by shahabganji**

### CODE REVIEW

File: Breaking_the_Monolith_Failed-Where_is_my-data.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Breaking the Monolith Failed! Where is my data?
+
+
+There's a myriad of complexity in breaking a monolith and modernize your products. There are countless books and
+articles helping how we could gradually replace the old functionalities and code bases with new one. Patterns like
+Strangler Fig, enables the software team to have control over how the old system will be finally shut down and the new
+system seamlessly fulfill old and new requirements. However, the core part of every software system is the data behind
+it. A software without data is like a joke without a punchline—leaves you hanging.
```
Comment (lailabougria) [inline Breaking_the_Monolith_Failed-Where_is_my-data.md]: This first paragraph sounds very general and a bit artificial, I think it's important to connect to a problem that users are facing, following your title. Things get more interesting when you start talking about the strangler fig pattern. Try to ask yourself: when you think of your audience, what problem are they facing today that they want to solve, which would make them want to come to your talk?

File: Breaking_the_Monolith_Failed-Where_is_my-data.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Being able to liberate the data in legacy applications and integrate them into the new systems is just as important as
+changing the code base and how the components of system are organized; having a modern and flexible data model serving
+the new requirements of the system requires to know how we could untangle our new design from the old model.
```
Comment (lailabougria) [inline Breaking_the_Monolith_Failed-Where_is_my-data.md]: There are many sentences here that make the same point, it reads a bit duplicated. 
I would suggest making this part of the first paragraph and condensing it.

In the second paragraph, what I'm looking for is how you ran into these challenges and how you've addressed them, eg. Put yourself into the audience's shoes and ask yourself, why should I listen to the speaker? What do they bring that can help me move forward?

File: Breaking_the_Monolith_Failed-Where_is_my-data.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+We will examine different patterns that can be leveraged to liberate data in your system and make them
+flow in the veins of it. We will examine the advantages and disadvantages of each approach, talk about tools we could
+employ to achieve this goal. By the end of this session, we will be better equipped to make informed decisions about how
+to migrate and synchronize data in our software applications.
```
Comment (lailabougria) [inline Breaking_the_Monolith_Failed-Where_is_my-data.md]:
```suggestion
Throughout this session, we will examine different patterns that can be leveraged to liberate data in your system and allow it to flow seamlessly. We will examine the advantages and disadvantages of each approach and discuss tools we could employ to achieve this goal. By the end of this session, you will be better equipped to make informed decisions about how
to migrate and synchronize data in your software applications.
```

**Review by lailabougria — COMMENTED** [review]
Hi Shahab,

My main concern with the abstract is that it's not clear to me what I will get from this session. What main issue are we tackling here? Would I join to learn about the strangler fig pattern in general? Or is it more about data, as the title suggests? I left a few comments inline as to what I'm looking for, I hope that's helpful to you.

What tends to help for me is putting myself into the audience shoes. If this was me before I had all this knowledge, what would I want to read here to come to this session?

File: Breaking_the_Monolith_Failed-Where_is_my-data.md (hunk 2 above)
Comment (shahabganji) [reply Breaking_the_Monolith_Failed-Where_is_my-data.md]: > In the second paragraph, what I'm looking for is how you ran into these challenges and how you've addressed them

Should I mention specific examples? Or just telling a story that we have an issue and we solved it using some patterns? 

--- 

I think this is exactly my issue, that I don't know whether I should name any specific approach like CDC + Debezium or just keep it general with for instance data liberation patterns?

**Comment by shahabganji** [comment]
Hi @lailabougria 

I've tried to make some changes based on your feedback, I hope it got better! Any idea?

**Comment by lailabougria** [comment]
I think the abstract has strongly improved now. Maybe it could be shortened a little, but I think it's a huge step forward versus the first version.
FYI: I've only re-reviewed [this one](https://github.com/shahabganji/abstracts/blob/139cd772a5c226049984f37f65da961abea54f05/Breaking_the_Monolith_Failed-Where_is_my-data.md)

**Comment by shahabganji** [comment]
Hi @lailabougria 

Thanks for the review, I know that one was what you helped me with! :)

**Pull request merged**